extensions/azure/vault/src/main/java/org/eclipse/dataspaceconnector/security/azure/AzureVaultExtension.java

```diff
@@ -55,28 +55,31 @@ public CertificateResolver getCertificateResolver() {
     }
 
     @Override
-    public void intializeVault(ServiceExtensionContext context) {
+    public void initializeVault(ServiceExtensionContext context) {
         String clientId = context.getSetting("edc.vault.clientid", null);
```

Comment:
Since you're at it: can you extract constants for the properties and annotate them with `@EdcSetting`?

Reply:
Done!